gradle.properties

```diff
@@ -5,4 +5,4 @@ group = com.enonic.app
 projectName = metafields
 appName = com.enonic.app.metafields
 xpVersion = 7.0.0
```

Comment:
We need to upgrade `xpVersion` to 7.7.0, otherwise people will be able to install this on versions < 7.7.0 and won't see the counter.

Reply:
Will this break if run on a version earlier then 7.7 ? I hoped that any future improvements could be usefull on <7.7

Reply:
It won't break but, unless there are other improvements in this version, it's meaningless to release a sole feature implemented in 7.7 without upgrading xpVersion to 7.7. If there are other improvements as well then it's ok, but we have to specify in the release notes that counter requires upgrade to XP 7.7 and CS 3.3